Danki2/Assets/Scripts/Stats/StatsManager.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class StatsManager
+{
+    private readonly Stats baseStats;
+    private readonly Stats frameStats;
+
+    public StatsManager(Stats baseStats)
+    {
+        this.baseStats = baseStats;
+        this.frameStats = new Stats(this.baseStats);
+    }
+
+    public int this[Stat stat]
```

Comment:
I've never seen this syntax before - cool

Reply:
Yeah it was new to me too